reactioncommerce/reaction
# Use JSON format for trace log level

Resolves #1398

To test:
- Nothing to see on the UI, since this is about logging
- Start the app using `reaction`, confirm that the usual logs still work as before
- Restart the app with `REACTION_LOG_LEVEL` set as `TRACE`, check that trace logs appear as raw json. (you'll get loads of logs, as you'll now be seeing all debug and trace logs).

NB: Setting log format for `trace` as json, makes Bunyan use same for `debug` (so debug logs also appear as json).
